### PR TITLE
NTR Nasal serous secreting cells

### LINKED ISSUE #2316: [NTR-cxg] nasal serous secreting cell
Please check that the term does not already exist by using the ontology search tool OLS:
https://www.ebi.ac.uk/ols4/ontologies/cl

I have not found extensive information on the exact location of nasal serous secreting cells, I just decided to include "nasal cavity" only. Suggestions are welcome. 

**IMPORTANT**: This cell has been identified from a CellxGene [dataset](https://cellxgene.cziscience.com/e/066943a2-fdac-4b29-b348-40cede398e4e.cxg/) through a knowledge graph analysis. Tracker for novel cell terms identified from CxG dataset through knowledge graphs can be found [here](https://docs.google.com/spreadsheets/d/1mZWx7Wuh0yWrAMIGPqiI_6Wpu_ZoaCHtbcCV3opbEYs/edit?pli=1#gid=0).

**Preferred term label**
nasal serous secreting cell

**Synonym(s)** (add reference(s), please)
SMG serous (nasal)

**Definition** (free text, with reference(s), please. PubMed ID format is PMID:XXXXXX)
A serous secreting cell that is part of the nasal cavity.

**Parent cell type term** (check the hierarchy here https://www.ebi.ac.uk/ols4/ontologies/cl)
[serous secreting cell](http://purl.obolibrary.org/obo/CL_0000313)

**Anatomical structure where the cell type is found** (check Uberon for anatomical structures: https://www.ebi.ac.uk/ols4/ontologies/uberon)
[nasal cavity](http://purl.obolibrary.org/obo/UBERON_0001707)

**Your ORCID**
https://orcid.org/0000-0002-0098-8958

**Additional notes or concerns** If there are markers that uniquely identify the cell type in a particular species, they can be added here.
The CellxGene dataset found to identify this cell type is from this paper [PMID:37291214](https://pubmed.ncbi.nlm.nih.gov/37291214)

### PR DESCRIPTION
**Pull request by AvolaAmg**
Fixes #2316

### CODE REVIEW

**Review by aleixpuigb — CHANGES_REQUESTED** [review]
Please, see inline comments.

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -32194,6 +32195,16 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30382198") oboInOwl:hasE
...
 
+# Class: obo:CL_4042014 (nasal serous secreting cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37291214") obo:IAO_0000115 obo:CL_4042014 "A serous secreting cell that is part of the nasal cavity.")
+AnnotationAssertion(terms:contributor obo:CL_4042014 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(terms:date obo:CL_4042014 "2024-07-12T10:58:06Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4042014 "https://cellxgene.cziscience.com/e/9f222629-9e39-47d0-b83f-e08d610c7479.cxg/")
+AnnotationAssertion(rdfs:label obo:CL_4042014 "nasal serous secreting cell"@en)
+SubClassOf(obo:CL_4042014 obo:CL_0000313)
+SubClassOf(obo:CL_4042014 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0011148))
```
[the remark this answers is not in the document]
Comment (dosumis) [reply src/ontology/cl-edit.owl:32206]: I agree - it looks like these cells are in the nasal serous/seromucous glands 

https://www.perplexity.ai/search/tell-me-about-the-structure-an-5aSwHXGcQEuo05I.VpnwaA
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1468122/

[nasal cavity respiratory epithelium](http://purl.obolibrary.org/obo/UBERON_0005385) is close and can do for now I think, until we have a better option (i.e. sufficient to close this ticket)/

But I also think we need to extend Uberon

image
image

The terms in Uberon don't reflect the parent class.

"olfactory gland" = Bowman's gland - which is in the olfactory epithelium.
I think we need serous and seromucous glands & these need a relationship to [nasal cavity respiratory epithelium](http://purl.obolibrary.org/obo/UBERON_0005385) 

We also need a standard pattern for gland partonomy - relationship to mucosa terms plus glandular epithelia & relationship to the epithelia with which they are continuous.  This is a very general issue for all mucosa including lung and gut, so we need some general patterns.

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -32197,12 +32197,14 @@ SubClassOf(obo:CL_4042013 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFr
 
 # Class: obo:CL_4042014 (nasal serous secreting cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37291214") obo:IAO_0000115 obo:CL_4042014 "A serous secreting cell that is part of the nasal cavity.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37291214") obo:IAO_0000115 obo:CL_4042014 "A serous secreting cell that is part of the nasal cavity respiratory epithelium. This cell is part of some submucosal gland.")
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:32200]: Can you change `some` for `a`? It reads a bit better.

Comment (AvolaAmg) [reply src/ontology/cl-edit.owl:32200]: of course! on it

**Review by aleixpuigb — APPROVED** [review]
Looks good to me!

**Comment by aleixpuigb** [comment]
#gogoeditdiff

**Review by aleixpuigb — APPROVED** [review]
Conflicts have been resolved

**Pull request merged**